test/test_pipeline/components/feature_preprocessing/test_select_rates_regression.py

```diff
@@ -0,0 +1,86 @@
+import unittest
+
+import numpy as np
+import scipy.sparse
+import sklearn.preprocessing
+
+from autosklearn.pipeline.components.feature_preprocessing.select_rates_regression import \
+    SelectRegressionRates
+from autosklearn.pipeline.util import _test_preprocessing, get_dataset
+
+
+class SelectRegressionRatesComponentTest(unittest.TestCase):
+    def test_default_configuration(self):
+        transformation, original = _test_preprocessing(SelectRegressionRates)
+        self.assertEqual(transformation.shape[0], original.shape[0])
+        self.assertEqual(transformation.shape[1], 4)
+        self.assertFalse((transformation == 0).all())
+
+        transformation, original = _test_preprocessing(
+            SelectRegressionRates, make_sparse=True)
+        self.assertTrue(scipy.sparse.issparse(transformation))
+        self.assertEqual(transformation.shape[0], original.shape[0])
+        self.assertEqual(transformation.shape[1], int(original.shape[1] / 2))
+
+        # Custom preprocessing test to check if clipping to zero works
```

Comment:
I'm not sure if I understand that test. Where is there some clipping to zero going on?